cloudaux/aws/iam.py

```diff
@@ -165,11 +170,11 @@ def get_role_instance_profiles(role, client=None, **kwargs):
 
     return [
         {
-            'path': ip['Path'],
-            'instance_profile_name': ip['InstanceProfileName'],
-            'create_date': ip['CreateDate'].strftime('%Y-%m-%dT%H:%M:%SZ'),
-            'instance_profile_id': ip['InstanceProfileId'],
-            'arn': ip['Arn']
+            'Path': ip['Path'],
+            'InstanceProfileName': ip['InstanceProfileName'],
+            'CreateDate': ip['CreateDate'].replace(tzinfo=None, microsecond=0).isoformat() + 'Z',
```

Comment:
This datetime formatting routine is introduced about 10 times throughout this change set. Is there room for a utility module of some kind in cloudaux?

Reply:
Maybe.

Reply:
Done.